package.json

```diff
@@ -73,6 +73,8 @@
     "pathval": "~0.1.1",
     "prompt": "~0.2.14",
     "strip-json-comments": "~1.0.2",
+    "to-double-quotes": "^1.0.0",
```

Comment:
Ah, I changed my default to tilde. I guess we'll opt for caret first and then discuss whether or not it should be a tilde during PR review?